### PR TITLE
[lodepng] Update to commit e34ac0@2020-03-15

### PR DESCRIPTION
**Pull request by BurningEnlightenment**
I'm currently preparing a cmake integrated bgfx port (requested here #1303) which requires a newer lodepng version. Additionally I added proper cmake import configs, because otherwise the debug library build cannot be found (the release binary always takes precedence).

### CODE REVIEW

File: ports/lodepng/portfile.cmake [hunk 1]
```diff
@@ -25,7 +24,12 @@ vcpkg_configure_cmake(
 vcpkg_install_cmake()
 vcpkg_copy_pdbs()
 
+# Moves all .cmake files from /debug/share/lodepng/ to /share/lodepng/
+vcpkg_fixup_cmake_targets()
+
 vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/lodepng)
 
 
 file(INSTALL ${SOURCE_PATH}/lodepng.h DESTINATION ${CURRENT_PACKAGES_DIR}/share/lodepng RENAME copyright)
```
Comment (Cheney-W) [inline ports/lodepng/portfile.cmake]: Could you please modify /share/lodepng as /share/${PORT} ? Thank you.

File: ports/lodepng/portfile.cmake [hunk 2]
```diff
@@ -25,7 +24,12 @@ vcpkg_configure_cmake(
...
+
+vcpkg_test_cmake(PACKAGE_NAME lodepng)
```
Comment (Cheney-W) [inline ports/lodepng/portfile.cmake]: The vcpkg_test_cmake() function has been removed from VCPKG, so could you please remove this sentence from your change? Thank you.

**Review by Cheney-W — CHANGES_REQUESTED** [review]

File: ports/lodepng/portfile.cmake (hunk 2 above)
Comment (BurningEnlightenment) [reply ports/lodepng/portfile.cmake]: Done. However, I'd like to point out that `.\vcpkg create` still adds that function call to all newly generated projects.

Comment (Cheney-W) [reply ports/lodepng/portfile.cmake]: Thanks for pointing it out. We will consider updating this feature.

**Review by Cheney-W — APPROVED** [review]

**Pull request merged**